oracle-jdk-installer/linux/8/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+
+COPY jdk-8u401-linux-x64.tar.gz oracle-jdk.tar.gz
```

Comment:
Just in case, could you please add a new line here?

Reply:
Thank you for the comments, I revised in https://github.com/scalar-labs/docker/pull/34/commits/326e56e63b84543eab95c840ae29b0bd7aed839c